docs: Add policy examples page (#16072)
hashicorp/consul: website/content/docs/security/acl/acl-policies.mdx

This topic describes policies, which are components in Consul's access control list (ACL) system. Policies define which services and agents are authorized to interact with resources in the network.

> Looking for an policy example for your use case? Refer to the [Policy Examples](/consul/docs/security/acl/acl-policy-examples) topic.

Reviewer: ```suggestion
Refer to the [Policy Examples](/consul/docs/security/acl/acl-policy-examples) for example policy configurations.
```
Rhetorical and call-and-response type questions are prohibited per the style guide.

The `global-management` policy grants unrestricted privileges to any token linked to it. The policy is assigned the reserved ID of `00000000-0000-0000-0000-000000000001`. You can rename the global management policy, but Consul will prevent you from modifying any other attributes, including the rule set and datacenter scope.

> The following are the policy rules for the `global-management` policy.

Reviewer: ```suggestion
The following policy rules define the `global-management` policy.
```

Author: FWIW, I'm not sure how useful it is to show the policy rules for the built-in global-management and namespace-management policies. I don't think users will need to reference policy rules for built-in policies that often, and the policies are long and clutter these sections. 

I included them because I tracked down the rules, and for completeness, and because I like the docs having many examples of working policies to reference.

I thought making the policy rules collapsable/expandable sections would be a good compromise, but wasn't sure how to do a collapsable section.

  policy = "write"
  intentions = "write"
}
session_prefix "" {
  policy = "write"
}
```

</Tab> 

<Tab heading="ENT">

In Consul Enterprise, the `global-management` policy contains all of the OSS rules and
adds additional partition and namespace scoped rules.

```hcl
acl = "write"
agent_prefix "" {
  policy = "write"
}
event_prefix "" {
  policy = "write"
}
key_prefix "" {
  policy = "write"
}
keyring = "write"
node_prefix "" {
  policy = "write"
}
operator = "write"
mesh = "write"
peering = "write"
query_prefix "" {
  policy = "write"
}
service_prefix "" {
  policy = "write"
  intentions = "write"
}
session_prefix "" {
  policy = "write"
}

partition_prefix "" {
  mesh = "write"
  peering = "write"
  namespace "default" {
    node_prefix "" {
      policy = "write"
    }
    agent_prefix "" {
      policy = "write"
    }
  }
  namespace_prefix "" {
    acl = "write"
    key_prefix "" {
      policy = "write"
    }
    node_prefix "" {
      policy = "read"
    }
    session_prefix "" {
      policy = "write"
    }
    service_prefix "" {
      policy = "write"
      intentions = "write"
    }
  }
}
```

</Tab> 

</Tabs>

### Namespace Management <EnterpriseAlert inline />

> The `namespace-management` policy will be injected into all namespaces you create. The policy will be assigned a randomized UUID and can be managed as a normal, user-defined policy within the namespace. 

Reviewer: ```suggestion
Consul injects the `namespace-management` policy into all namespaces you create. Consul also assigns the policy a randomized UUID that you can use to manage the policy within the namespace. 
```
Active voice and present tense

> The following are the policy rules for the namespace management policy. Because this

Reviewer: ```suggestion
The following policy rules define the namespace management policy. Because this
```
